server/src/main/scala/org/elasticmq/server/ElasticMQServer.scala

```diff
@@ -80,4 +83,40 @@ class ElasticMQServer(config: ElasticMQServerConfig) extends Logging {
       deadLettersQueue = cq.deadLettersQueue.map(dlq => DeadLettersQueueData(dlq.name, dlq.maxReceiveCount))
     )
   }
+
+  private def sortCreateQueues(cqs: List[config.CreateQueue]): List[config.CreateQueue] = {
```

Comment:
Looks nice! Two small things:

1. Maybe a comment on *why* we need to sort the queues would be helpful for future generations?
2. How about moving this method to `config` (so that `config` would have a `sortedQueues` method or such), or even moving it to a separate class and adding some basic tests (empty input list, one-element, not sorted)